test/congestion/congestion.csv

```diff
@@ -0,0 +1,100 @@
+62397298,62523814,20
```

Comment:
Could you add tests for invalid csv files / not numbers / negative numbers / more columns / `#header` etc.